onnx_chainer/export.py

```diff
@@ -343,17 +346,38 @@ def _export(model, args, filename, export_params, graph_name, save_text,
         graph, graph_name, input_tensors, output_tensors,
         initializer=initializers)
 
+    opset_imports = [helper.make_operatorsetid('', opset_version)]
+    if external_opset_imports is not None:
+        for domain, version in external_opset_imports.items():
+            opset_imports.append(helper.make_operatorsetid(domain, version))
     model = helper.make_model(
         onnx_graph,
         producer_name='Chainer',
         producer_version=chainer.__version__,
-        opset_imports=[helper.make_opsetid('', opset_version)]
+        opset_imports=opset_imports
     )
 
     model.ir_version = onnx.IR_VERSION
 
     rename_tensors(model)
-    checker.check_model(model)
+    try:
+        checker.check_model(model)
+    except Exception as e:
+        if external_converters is None:
+            raise e
+        else:
+            if not isinstance(
+                    e, onnx.onnx_cpp2py_export.checker.ValidationError):
+                raise e
+            elif 'No Op or Function registered' not in str(e):
+                # this error message is onnx>=1.4.0
+                raise e
+            else:
+                warnings.warn(
```

Comment:
Nice! I was thinking of just skipping `checker.check_model` when a custom converter is set, but I think your solution is much better!